### ISSUE TITLE
Error of "unsupported operand type(s) for -: 'list' and 'list'"

### ISSUE THREAD

**Issue by tdingquan** [microsoft/StemGNN#22]
When using min-max method on ECG data, we will encounter error of "unsupported operand type(s) for -: 'list' and 'list'".

Line 10, forecast_dataloader.py

**Comment by Fadegentle** [comment]
> 
Maybe the coder forgot to turn the list into `numpy.ndarray`, because he just use the `z_score` with `data = (data - mean) / std`. The `data` is `numpy.ndarray`, so it actually turns into `-` operand of  `numpy.ndarray`.

So you can try 
`scale = np.array(norm_statistic['max']) - norm_statistic['min'] + 1e-5`
instead of
`scale = norm_statistic['max'] - norm_statistic['min'] + 1e-5`

This is a useful trick.

**Comment by tdingquan** [comment]
Correct. I chose to turn the list into numpy.array before the `-` operation.
